docs/internal/README.md

```diff
@@ -1,27 +1,40 @@
 # Internal documentation
 
-This documentation covers the internal structure of r. Everything here is no part of the stable API and might change completely in the future.
+This documentation covers the internal structure of marker.
+Everything here is no part of the stable API and might change completely in the future.
 
 ## Structure
 
 ![](./marker-structure.png)
 
-This linter consists out of several crates, each with one specific job. The background color in the image indicates what a crate belongs to:
+This linter consists out of several crates, each with one specific job.
+The background color in the image indicates what a crate belongs to:
 
-* The yellow area is the stable API used by *lint_crates*. Additionally, it's expected, that the command interface of `cargo-marker` will remain constant.
+* The yellow area is the stable API used by *lint_crates*.
+Additionally, it's expected, that the command interface of `cargo-marker` will remain constant.
 * The red area is unstable and contains all driver independent crates and infrastructure.
-* The blue area is driver-specific. It'll use crates from the red area to create a stable AST representation and providing it to lint crates.
+* The blue area is driver-specific.
+It'll use crates from the red area to create a stable AST representation and providing it to lint crates.
 
-The jobs and stability guarantees are documented in the crate specific `README.md` files. This document will go over the step by step linting process for a linting process.
+The jobs and stability guarantees are documented in the crate specific `README.md` files.
+This document will go over the step by step linting process for a lint pass.
 
-1. `cargo-marker` starts off by verifying that a driver is available (In dev mode it will compile the driver for every run)
-2. `cargo-marker` compiles all defined lint crates and moves the binaries to a single target directory (currently `target/marker/lints`)
+1. `cargo-marker` starts off by verifying that a driver is available
+(In dev mode it will compile the driver for every run)
+2. `cargo-marker` compiles all defined lint crates
+and moves the binaries to a single target directory (currently `target/marker/lints`)
 3. `cargo-marker` sets environment values required by `marker_adapter`
-4. `cargo-marker` starts the driver. Currently, for rustc, this is done by setting `RUSTC_WORKSPACE_WRAPPER` and then calling `cargo check` which will in turn execute the driver like it was the usual rustc binary.
+4. `cargo-marker` starts the driver.
+Currently, for rustc, this is done by setting `RUSTC_WORKSPACE_WRAPPER`
+and then executing `cargo check` which will in turn execute the driver like it was the usual rustc binary.
```

Comment:
TIL, that lines following an item don't need to be indented o.O. I thought they needed to. Would you mind indenting them anyways, if they belong to an indented item?

Sorry, if this is too pedantic, this is the last thing :sweat_smile:  

```suggestion
4. `cargo-marker` starts the driver.
	Currently, for rustc, this is done by setting `RUSTC_WORKSPACE_WRAPPER`
	and then executing `cargo check` which will in turn execute the driver like it was the usual rustc binary.
```